tests/test_types_basic.py

```diff
@@ -533,7 +511,7 @@ def test_blank_hex(self):
         rv = self.cast(b'\\x')
         self.assertEqual(rv, b'')
 
-    def test_full_hex(self, upper=False):
+    def _test_full_hex(self, upper=False):
```

Comment:
Why was this needed? :)

Reply:
This was from a previous version. You're right it is no longer necessary and has been reverted in the latest revision.